aspnetcore-webapi/GettingStartedASPNETMongoDB/GettingStartedASPNETMongoDB/Controllers/CourseController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using GettingStartedASPNETMongoDB.Interfaces;
+using GettingStartedASPNETMongoDB.Models;
+
+namespace GettingStartedASPNETMongoDB.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CourseController : ControllerBase
+    {
+        private readonly ICourseService _CourseService;
+
+        public CourseController(ICourseService service)
+        {
+            _CourseService = service;
+        }
+
+        [HttpGet("{id:length(24)}")]
+        public async Task<ActionResult<Course>> GetById(string id)
+        {
+            var course = await _CourseService.GetById(id);
+
+            return course is null ? NotFound() : Ok(course);
```

Comment:
In most of the other null checks we are using `x == null`. We should seek to be consistent across the code with one version or the other. The `x is null` version is the more preferred style in modern .NET code, so I would recommend changing the others to match. Note that we can also use `x is not null` in the case where the check is `x != null`.